Classes/Controller/TimelineController.php

```diff
@@ -79,6 +82,15 @@ protected function initializeAction()
     }
 
     /**
+     * @param \TYPO3\CMS\Core\Page\PageRenderer $pageRenderer
+     * @return void
+     */
+    public function injectPageRenderer(\TYPO3\CMS\Core\Page\PageRenderer $pageRenderer)
```

Comment:
a) as you're importing the PageRenderer into the current namespace, you might not need the fully qualified namespace here
b) it might be worth trying the \@inject annotation

Reply:
I'll test and use @inject if it works at this point. As it's in controller context, it will not work as i recall it.